apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -29,19 +30,19 @@ export default function LearningGoal({
   studentLevelInfo,
 }) {
   const [isOpen, setIsOpen] = useState(false);
-
   const [isAutosaving, setIsAutosaving] = useState(false);
   const [autosaved, setAutosaved] = useState(false);
   const [errorAutosaving, setErrorAutosaving] = useState(false);
   const [learningGoalEval, setLearningGoalEval] = useState(null);
   const [displayFeedback, setDisplayFeedback] = useState('');
-  const understandingLevel = useRef(0);
+  const [displayUnderstanding, setDisplayUnderstanding] = useState(-1);
```

Comment:
nit: can default to `null` instead? I think it makes the conditionals below clearer if we can use `!displayUnderstanding`

Reply:
I'm using -1 because I was having an issue where the conditionals were recognizing 0 as a false value, but 0 is the value for "No Evidence," so it wasn't displaying whenever No Evidence was selected.

Reply:
ah got it. Can you extract the -1 into a constant? (more of a nit than a real concern)